index.html

```diff
@@ -2303,6 +2355,23 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
+          </dt>
+          <dd>
+            <p>
+              The UA's best estimation in seconds of audio output latency, i.e.
+              the delay between the moment the implementation sends a buffer to
+              the system and the audible sound from the audio output (speaker
+              or connector).
```

Comment:
I think we have some language in the "Processing Model" section of the spec to refer to the rendering of audio, but I can't remember.